specs/blocks.spec.ts

```diff
@@ -305,6 +305,38 @@ test('attempts-exhausted response overrides answer-specific response', async ({
   await page.waitForSelector('#attemptsExhausted')
 })
 
+test('Math field component rendered', async ({ page }) => {
+  const htmlContent = `
+  <div class="os-raise-ib-pset" data-schema-version="1.0" data-retry-limit="3">
+    <div class="os-raise-ib-pset-problem" data-problem-type="input" data-solution='42' data-problem-comparator='math'>
+      <div class="os-raise-ib-pset-problem-content">
+        <p id="problem">MultipleChoice problem content: \\( x^2 \\)</p>
+      </div>
+      <div class="os-raise-ib-pset-encourage-response" data-answer="41">
+          <p>Almost there</p>
+      </div>
+      <div class='os-raise-ib-pset-attempts-exhausted-response'>
+          <p id="attemptsExhausted">The answer is 42</p>
+      </div>
+    </div>
+    <div class="os-raise-ib-pset-correct-response">
+      <p id="correct">Correct response with math: \\( x=2 \\)</p>
+    </div>
+    <div class="os-raise-ib-pset-encourage-response">
+      <p id="encourage">Encourage response with math: \\( x=2 \\)</p>
+    </div>
+  </div>
+  `
+  await mockPageContentRequest(page, htmlContent)
+  await page.goto('/')
+  await page.locator('.ML__content').click()
+  await page.waitForSelector('text=y')
```

Comment:
It took me a second, but I think the intent of this is to basically assert that the keyboard appears. If so, perhaps using `'.ML__keeyboard'` as the selector would make that a bit clearer + not subject to failure if we were to remove `y` from the keyboard :man_shrugging: ?

Reply:
Is this what you are ultimately suggesting between this comment and [this](comment)?

```js
await mockPageContentRequest(page, htmlContent)
  await page.goto('/')
  await page.locator('math-field').click()
  await page.waitForSelector('.ML__keyboard')
  await page.keyboard.type('42')
  await page.waitForSelector('text=42')
  await page.locator('text=Check').click()
  await page.waitForSelector('text=Correct')
```

Reply:
I think the reason Prabhdip did not use `math-field` as the locator is because when you click the `math-field` what you are actually clicking is a `<span>` inside of `math-field` with the class `.ML__content`. I tried making `math-field` the locator but the test failed when it tried to click the `math-field`.

The following test does pass:
```js

Reply:
:confused: -- I seem to get the following passing:

```javascript
  await mockPageContentRequest(page, htmlContent)
  await page.goto('/')
  await page.locator('math-field').click()
  await page.waitForSelector('.ML__keyboard')
  await page.keyboard.type('42')
  await page.waitForSelector('text=42')
  await page.locator('text=Check').click()
  await page.waitForSelector('text=Correct')
```

The output screenshot seems correct so it doesn't look like a false positive pass :thinking:  : 

![image](https://github.com/openstax/k12-apps-raise/assets/839025/ce5ba673-7ab7-4e0c-a6df-2a6c238c8262)

Reply:
Discussed live but noting here: It turns out the test was passing sometimes on @MReyna12 's machine, and the failures were due to it hitting a timeout. We're going to try the `'math-field'` selector and see if things pass on CI.

Reply:
Per a slack call with @rnathuji, we identified that while the test operates correctly, sometimes a timeout error is reached because it takes longer than 10000ms to complete the test. Per the documentation on `locator().click()` (found [here](https://playwright.dev/docs/api/class-locator#locator-click)), when all steps combined have not finished during the specified timeout (aka 10000ms), this method throws a [TimeoutError](https://playwright.dev/docs/api/class-timeouterror).